### PR TITLE
[V3] Code standards (black code format pass)

### PR DESCRIPTION
**Pull request by mikeshardmind**
### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes
ran black: code formatter against `redbot/` with `-l 99`

resulting code should be functionally equivalent via ast comparison

This works towards #1636 and the discussed goals in discord

### CODE REVIEW

File: redbot/cogs/admin/admin.py [hunk 1]
```diff
@@ -259,22 +252,20 @@ def pass_user_heirarchy_check(ctx: commands.Context,
...
-        await ctx.send("The announcement channel has been set to {}".format(
-            channel.mention
-        ))
+        await ctx.send("The announcement channel has been set to {}".format(channel.mention))
 
     @announce.command(name="ignore")
     @commands.guild_only()
     @checks.guildowner_or_permissions(administrator=True)
-    async def announce_ignore(self, ctx, *, guild: discord.Guild=None):
+    async def announce_ignore(self, ctx, *, guild: discord.Guild = None):
```
Comment (bobloy) [inline redbot/cogs/admin/admin.py:268]: Why does it add spaces around the `=` operator here and not below?

File: redbot/cogs/admin/announcer.py [hunk 2]
```diff
@@ -5,9 +5,8 @@
 
 
 class Announcer:
-    def __init__(self, ctx: commands.Context,
-                 message: str,
-                 config=None):
+
+    def __init__(self, ctx: commands.Context, message: str, config=None):
```
Comment (bobloy) [inline redbot/cogs/admin/announcer.py:9]: Here it does not add spaces around the `=` operator, despite doing it above frequently

**Review by bobloy — COMMENTED** [review]
Doesn't seem consistent.

Additionally, won't this likely cause unnecessary merge conflicts?

File: redbot/cogs/admin/admin.py [hunk 3]
```diff
@@ -40,18 +40,16 @@
...
-        self.conf.register_global(
-            serverlocked=False
-        )
+        self.conf.register_global(serverlocked=False)
 
         self.conf.register_guild(
             announce_ignore=False,
             announce_channel=None,  # Integer ID
-            selfroles=[]  # List of integer ID's
+            selfroles=[],  # List of integer ID's
```
Comment (Aioxas) [inline redbot/cogs/admin/admin.py:52]: Where did this even come from. Is that compliant?

File: redbot/cogs/admin/announcer.py (hunk 2 above)
Comment (tekulvw) [reply redbot/cogs/admin/announcer.py:9]: One of them has a type annotation and the other does not. It's consistent and compliant.

File: redbot/cogs/admin/admin.py (hunk 3 above)
Comment (tekulvw) [reply redbot/cogs/admin/admin.py:52]: Yes, it's compliant and a style choice.

**Review by tekulvw — DISMISSED** [review]

**Review by tekulvw — APPROVED** [review]

**Pull request merged**